src/index.html

```diff
@@ -9,10 +9,74 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/variables.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/stars.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/card.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/button.css"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="imac-photo">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac-photo"
+          width="160"
+          height="134"
+        />
+      </div>
+      <div>
+        <p class="product-card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+      </div>
+      <div class="product-card__code">
+        <p>Product code: 195434</p>
+      </div>
+      <div class="product-card__ratings">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="product-card__review">
```

Comment:
```suggestion
        </div>
        
        <div class="product-card__review">
```
Add one empty line between sibling blocks inside the same parent block for both of them.